pandas/io/parsers.py

```diff
@@ -244,8 +244,9 @@
     standard encodings
     <https://docs.python.org/3/library/codecs.html#standard-encodings>`_
 dialect : str or csv.Dialect instance, default None
-    If None defaults to Excel dialect. Ignored if sep longer than 1 char
-    See csv.Dialect documentation for more details
+    If provided, this parameter will override values for the following
+    parameters: `delimiter`, `doublequote`, `escapechar`, `skipinitialspace`,
+    `quotechar`, and `quoting`. See csv.Dialect documentation for more details.
```

Comment:
No, because a user could want the default values on purpose, and they should still know too.